mbed-os-env-cmake/Dockerfile

```
@@ -36,7 +36,7 @@ RUN pip3 install -U \
 
 # Set up mbed environment
 WORKDIR /root/
-RUN wget https://github.com/ARMmbed/mbed-os/raw/master/requirements.txt && \
+RUN wget https://github.com/ARMmbed/mbed-os/raw/master/tools/cmake/requirements.txt && \
```

```suggestion
RUN wget https://github.com/ARMmbed/mbed-os/raw/latest/tools/cmake/requirements.txt && \
```
@0xc0170 @donatieng What do you think? Or is `latest` not recommended and for backward compatibility of the old tools only?

---

I'd recommend matching the version of Mbed OS and the version of the Dockerfile, or at least putting the Mbed OS version into the Dockerfile's version tag/string. So, if a Dockerfile says it works for Mbed OS 7.2.3, the version of requirements.txt it used would have been from version Mbed OS 7.2.3 from Mbed OS.

`latest` won't work if we want to use this image in CI to test the next release of Mbed OS and Mbed OS's requirements changed since the previous release.

---

latest would be an answer to have released version always. But then a question is what tools we should install ? We fetch any the latest (major versions could break the build).

---

github.com/ARMmbed/mbed-os-docker-images/pull/6 does also to latest :-)

---

I agree with @Patater 

I will do that as part of https://github.com/ARMmbed/mbed-os-docker-images/pull/7 Once done, I will close this PR.

@pan- @donatieng @0xc0170  please shout if that's not alright.